classes/navigationMenu/NavigationMenuItemDAO.inc.php

```diff
@@ -124,6 +124,25 @@ public function getByTypeAndTitleLocaleKey($contextId, $menuItemType, $menuItemT
 		return $returner;
 	}
 
+	/**
+	 * Retrieve the menu items with the specified type.
+	 * @param $type int NMI_TYPE_...
+	 * @param $contextId int
+	 * @return DAOResultFactory containing matching NavigationMenuItems
+	 */
+	function getByType($type, $contextId = null) {
+		$params = array($type);
+		if ($contextId == null) {
```

Comment:
Did you mean `$contextId !== null` here and below?

Reply:
Yup :-P